interface/billing/ub04_form.php

```diff
@@ -39,10 +39,10 @@
 <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
 <meta charset="utf-8" />
```

Comment:
hi @sjpadgett ,
I haven't been able to test this script since can't get the wkhtmltopdf to work. I am installing it via composer and it is there in the right place. Then when I try to create the ub04 pdf:

![image](https://user-images.githubusercontent.com/278968/75852962-f92fd200-5da1-11ea-87c4-be9cc50b9ad5.png)

I then get the following error:
![image](https://user-images.githubusercontent.com/278968/75852994-06e55780-5da2-11ea-87bd-2fc71b86efe7.png)

And i did confirm that /var/www/localhost/htdocs/openemr/vendor/openemr/wkhtmltopdf-openemr/bin/linux/wkhtmltopdf64-linux was there and gave it all possible permissions without success

Reply:
sorry @bradymiller @im-Amitto Just saw this:) Most likely file version to o.s issue. I'll need to probably update repo and maybe put on packagist. Will let you know soon.